packages/@ama-sdk/core/src/fwk/date.ts

```diff
@@ -110,16 +100,10 @@ export namespace utils {
   }
 
   export class DateTime extends CommonDate {
-
-    constructor()
-    constructor(value: number)
-    constructor(value: string)
-    constructor(date: _NativeDate)
-    constructor(dateTimeUtils: DateTime)
-    constructor(dateUtils: Date)
-    constructor(otherDate: any)
```

Comment:
breaking ?

Reply:
I still have a definition matching all of them 
I just regroup them

Reply:
you no longer have a definition that matches `constructor(otherDate: any)`

Reply:
nevermind `constructor(...args: any[])` should match